newsfragments/README.md

```diff
@@ -8,15 +8,15 @@ relevant to people working on the code itself.)
 Each file should be named like `<ISSUE>.<TYPE>.rst`, where
 `<ISSUE>` is an issue number, and `<TYPE>` is one of:
 
-* `breaking`
```

Comment:
Were these changes all related to pre-commit checks?

Reply:
Yup, that's the `mdformat` tool I added. Once this PR is in, I'll make one to close https://github.com/ethereum/ethereum-python-project-template/issues/83